test/uk/gov/hmrc/cdsreimbursementclaimfrontend/connectors/HttpSupport.scala

```diff
@@ -100,7 +100,7 @@ trait HttpSupport { this: MockFactory with Matchers ⇒
 
   def mockPost[A](url: String, headers: Seq[(String, String)], body: A)(
     result: Option[HttpResponse]
-  ): Unit =
+  ): Any =
```

Comment:
why do you need to change the type of the return value here?

Reply:
When I compile with intellij this fails because you don't actually return Unit here, but something else

Reply:
OK, I'll check that later. It's never been a problem elsewhere so that's strange